payloads/external/edk2/Makefile

```diff
@@ -137,6 +137,10 @@ endif
 ifneq ($(CONFIG_EDK2_DISABLE_MTRR_PROGRAMMING),y)
 BUILD_STR += -D DISABLE_MTRR_PROGRAMMING=FALSE
 endif
+# DASHARO_USB_CONFIG	= FALSE
```

Comment:
Already present in line: https://github.com/Dasharo/coreboot/blob/2344a7c8fecc38bfdecd60f05727d463c8bcfad2/payloads/external/edk2/Makefile#L155

Reply:
as above